R/pkg/R/DataFrame.R

```diff
@@ -767,6 +767,14 @@ setMethod("repartition",
 #'                      using \code{spark.sql.shuffle.partitions} as number of partitions.}
 #'}
 #'
+#' At least one partition-by expression must be specified.
```

Comment:
this won't be formatted correctly in R doc due to the fact that "empty line" is significant. L769 should be removed to ensure it is in description

Reply:
I see. What about on 761? I see several docs around here with empty lines (829, 831 below). Are those different? These comments are secondary, but I guess they belong in the public docs as much as anything.

Reply:
761 is significant also, but correct. 

essentially:
1. first line of the blob is the title (L760)
2. second text after "empty line" is the description (L762)
3. third after another "empty line" is the "detail note" which is stashed all the way to the bottom of the doc page

so generally you want "important" part of the description on top and not in the "detail" section because it is easily missed. 

this is the most common pattern in this code base. there's another, where multiple function is doc together as a group, eg. collection sql function (in functions.R). other finer control is possible as well but not used today in this code base.

similarly L829 is good, L831 is a bit fuzzy - I'd personally prefer without L831 to keep the whole text in the description section of the doc.

Reply:
@felixcheung have a look at https://github.com/apache/spark/pull/23167

Reply:
@felixcheung Thanks, I did not know about this strict doc formatting rule in R.

@srowen Thanks for taking care of the fix!